src/features/links/screens/CollectionDetailScreen.tsx

```
@@ -0,0 +1,273 @@
+        </Text>
+      </View>
+    );
+  }
+
+  const renderEmpty = useCallback(
+    () => (
+      <View className="items-center px-8 py-12">
+        <View className="mb-6 rounded-full bg-slate-50 p-6">
+          <FolderOpen size={48} color={colors.iconMuted} strokeWidth={1} />
+        </View>
+        <Text className="mb-2 text-center text-lg font-semibold text-slate-800">
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)